_chapters/js/dependencies.md

```diff
@@ -22,6 +22,35 @@ It's not needed and should be phased out and replaced by npm. More information c
 
 npm installs its dependencies in the `node_modules` directory. Common conventions dictate that `node_modules` should be excluded from source control by adding it to your project's `.gitignore`, primarily because Node.js-friendly environments (such as 18F's deployment service, [Cloud Foundry], and other such as [Heroku]) recognize the existence of `package.json` and automatically install dependencies as needed.
 
+### Install npm
+
+It's recommended that developers install using [nvm](https://github.com/creationix/nvm). nvm (which stands for Node version manager) is a software that allows you to run multiple versions of node in different projects on the same computer. It's benefits include
```

Comment:
The section title is "Install npm" but the first weird three letter combination starting with "n" is `nvm`. I might change the first sentence to read:

> We recommend that developers (note 1) install both `node` and `npm` through a tool called `nvm`. `nvm` (which stands for node version manager)....

Notes:
1. Should we use something more generic, like folks? All sorts of people from across 18F will probably have to install `npm` to get various build tasks working.